test/Interop/Cxx/class/method/methods-addressable-silgen.swift

```diff
@@ -0,0 +1,17 @@
+
+// RUN: %target-swift-emit-silgen -I %S/Inputs -enable-experimental-feature AddressableParameters -enable-experimental-cxx-interop %s | %FileCheck %s
+
+import Methods
+
+// REQUIRES: swift_feature_AddressableParameters
+
+public func addressableTest(x: borrowing @_addressable NonTrivialInWrapper) {
+  let m = HasMethods()
+  m.nonTrivialTakesRef(x)
+  // No copy from the argument to the apply.
+  // CHECK: bb0(%0 : @noImplicitCopy $*NonTrivialInWrapper):
+  // CHECK: %1 = copyable_to_moveonlywrapper_addr %0
+  // CHECK: %3 = mark_unresolved_non_copyable_value [no_consume_or_assign] %1
+  // CHECK: %14 = moveonlywrapper_to_copyable_addr %3
+  // CHECK: %16 = apply %15(%14, %12) : $@convention(cxx_method) (@in_guaranteed NonTrivialInWrapper, @in_guaranteed HasMethods) -> ()
```

Comment:
Can these registers be captured via `[[#REGNAME:]]`, so that the test is resilient to register numbers?

Reply:
Good point, updated.